### PR TITLE
Fixed bug where multiple configs introduces undefined in the merged configs for a certain module

### PR DESCRIPTION
**Pull request by nkimaina**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "clean": "rimraf dist",
     "start": "webpack-dev-server",
+    "start:https": "webpack-dev-server --https",
```
Comment (brandones) [inline package.json]: I think the convention in our other modules is to call this `start-https`

**Comment by brandones** [comment]
Interesting! Could you write a test for this? I'm not totally sure what the problem is.

**Comment by nkimaina** [comment]
I have updated the unit test to reflect the test cases that produces this bug. I didn't the description clear sorry. It occurs when you have multiple configs defined. 
The unit test in question is:  
```npx jest -t 'works for multiple modules'```
The following screen shots will show you the bugs:
Screenshot 2020-02-24 at 11 07 47

Screenshot 2020-02-24 at 10 40 17

Screenshot 2020-02-24 at 11 40 52

That said. I have added the test scenario that produces it. 
See the example in our ampath repo:  
https://github.com/AMPATH/ampath-esm-patient-chart-widgets/pull/3/commits/eb23cad12be83096259b4997884ff74c6bb2a471

File: src/module-config/module-config.test.ts [hunk 2]
```diff
@@ -118,6 +119,101 @@ describe("getConfig", () => {
     await expect(fooConfig).resolves.toHaveProperty("foo", "qux");
     const barConfig = Config.getConfig("bar-module");
     await expect(barConfig).resolves.toHaveProperty("bar", "baz");
+
+    // TEST CASE 2: One that bugs out for multiple modules: see conversation on:
+    // https://github.com/openmrs/openmrs-esm-module-config/pull/18
+    Config.clearAll();
+    Config.defineConfigSchema("@openmrs/esm-patient-chart", {
```
Comment (brandones) [inline src/module-config/module-config.test.ts]: This is a very complex example of the problem. It doesn't make it clear what exactly the issue is. Could you please reduce it to a [minimal working example](https://stackoverflow.com/help/minimal-reproducible-example)?

**Comment by nkimaina** [comment]
The problem is the following line ends up with [ object, undefined]. It's easier to fix this here by eliminating the undefined (and end up with [object]) than try to fix the consequences downstream. 
I am not able to login into our distro because the bug breaks the app. 
https://github.com/openmrs/openmrs-esm-module-config/blob/master/src/module-config/module-config.ts#L108

**Comment by brandones** [comment]
Thanks for finding and fixing this!

**Pull request merged**